charts/argocd-apps/values-integration.yaml

```diff
@@ -991,6 +991,21 @@ applications:
           secretKeyRef:
             name: rails-secret-key-base
             key: SECRET_KEY_BASE
+- name: license-finder
+  helmValues:
+    appImage: 
```

Comment:
Looks like this line was accidentally left in, maybe? You don't need to specify `appImage` and the missing value here is causing a syntax error, I think. (Similarly in `values-test.yaml`.)
```suggestion
```